Accordion/themes/Accordion_template.less

```diff
@@ -0,0 +1,106 @@
+@import "../../../../delite/themes/bootstrap/includes";
+
+.d-accordion {
```

Comment:
detail: it should be tab indented I guess, although we do have .less files that aren't elsewhere...